### PR TITLE
add phase one keyboard

### PR DESCRIPTION
**Pull request by moyi4681**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kbdfans/phaseone/rules.mk [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
```
Comment (fauxpark) [inline keyboards/kbdfans/phaseone/rules.mk:17]:
```suggestion
```

File: keyboards/kbdfans/phaseone/phaseone.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#include "phaseone.h"
+void keyboard_pre_init_kb(void) {
+    setPinOutput(D4);
```
Comment (fauxpark) [inline keyboards/kbdfans/phaseone/phaseone.c:19]: What is this for?

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kbdfans/phaseone/phaseone.c (hunk 2 above)
Comment (moyi4681) [reply keyboards/kbdfans/phaseone/phaseone.c:19]: D4 connect to one LED that always on

**Comment by moyi4681** [comment]
can you guys check and approve this pull request again, thanks @fauxpark @drashna

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**